### PR TITLE
Add /shrug command

### PR DESCRIPTION
**Pull request by geekgonecrazy**
@RocketChat/core 

I found my self wanting to use the shrug command a few times, and thought about creating this.  Then discord recently added it.  So I figured why not?

image
Results in:
image

### CODE REVIEW

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+* @param {Object} message - The message object
+*/
+
+
+function Shrug (command, params, item) {
+	if (command === "shrug") {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯\\_(ツ)_/¯';
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l12)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('shrug', Shrug, {
+	description: 'Slash_Shrug_Description',
+	params: 'your message (optional)'
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l19)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	if (command === "shrug") {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯\\_(ツ)_/¯';
+		Meteor.call('sendMessage', msg);
+	}
+}
+
+RocketChat.slashCommands.add('shrug', Shrug, {
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l17)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+	description: 'Slash_Shrug_Description',
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l18)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+*/
+
+
+function Shrug (command, params, item) {
+	if (command === "shrug") {
+		var msg;
+
+		msg = item;
+		msg.msg = params + ' ¯\\_(ツ)_/¯';
+		Meteor.call('sendMessage', msg);
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l13)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+/*
+* Shrug is a named function that will replace /shrug commands
+* @param {Object} message - The message object
+*/
+
+
+function Shrug (command, params, item) {
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Unexpected space before function parentheses.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l7)

File: packages/rocketchat-slashcommands-shrug/shrug.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+	if (command === "shrug") {
```
Comment (rodrigok) [inline packages/rocketchat-slashcommands-shrug/shrug.js]: Codacy Issue found: [Strings must use singlequote.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2530061390/issues/source?bid=3199634&fileBranchId=3283147#l8)

**Comment by Molle500** [comment]
If anything /tableflip should be added :) : 
(╯°□°）╯︵ ┻━┻

**Pull request merged**